### PR TITLE
Add ResultUtil class

### PR DESCRIPTION
**Pull request by grevend-bosch**
Signed-off-by: David Greven <fixed-term.David.Greven@de.bosch.com>

### CODE REVIEW

**Comment by grevend-bosch** [comment]
@sbckr Coverage will be resolved later in a different pull request.

File: src/main/java/io/carbynestack/cli/util/ResultUtil.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    /**
+     * Returns a {@link Success} instance.
+     *
+     * @param value the success value
+     * @param <S>   the success value type
+     * @param <F>   the failure reason type
+     * @return the created {@code Success} instance
+     * @since 0.7.0
+     */
+    public static <S, F> Success<S, F> succ(S value) {
```
Comment (sbckr) [inline src/main/java/io/carbynestack/cli/util/ResultUtil.java:33]: I personally find it strange to name / abbreviate the factory method for Success with succ

Comment (grevend-bosch) [reply src/main/java/io/carbynestack/cli/util/ResultUtil.java:33]: @sbckr Succ seems to be the official abbreviation for Success. If you have a better proposal that is shorter than Success, please let me know.

Comment (grevend-bosch) [reply src/main/java/io/carbynestack/cli/util/ResultUtil.java:33]: Maybe scs?

Comment (sbckr) [reply src/main/java/io/carbynestack/cli/util/ResultUtil.java:33]: why not simply _success_? however, _scs_ would be okay too

**Review by sbckr — APPROVED** [review]
LGTM

**Pull request merged**